package.json

```diff
@@ -88,6 +88,7 @@
     "node-persist": "^3.1.3",
     "openai": "^4.20.0",
     "play-sound": "^1.1.6",
+    "pocketbase": "^0.19.0",
```

Comment:
@Blankeos please add pocketbase in the stacks/package.json

Reply:
@merwanehamadi noted on this, just force-pushed a change based on revisions.